tests/testthat/testImportSNV.TCGA.R

```diff
@@ -0,0 +1,181 @@
+# testImportSNV.TCGA.R
```

Comment:
In all of your function calls, turn logging off all the time, except in one block where you check the log.